bitrise.yml

```diff
@@ -890,7 +863,7 @@ workflows:
             set -x
             gem install fastlane -v 2.219.0
             fastlane -version
-            ./firefox-ios/l10n-screenshots.sh --test-without-building $MOZ_LOCALES
+            ./firefox-ios/l10n-screenshots.sh $MOZ_LOCALES
```

Comment:
we are using a build and then test, if we build for each local, the time it takes to run this is going to increase considerably